src/main/java/com/weddineo/authentication/service/RegistrationService.java

```diff
@@ -0,0 +1,29 @@
+package com.weddineo.authentication.service;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.UserRecord;
+import com.google.firebase.auth.UserRecord.CreateRequest;
+import com.weddineo.authentication.exception.RegistrationRuntimeException;
+import com.weddineo.authentication.model.RegistrationInfo;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RegistrationService {
+
+    public UserRecord createUser(RegistrationInfo registrationInfo){
+        CreateRequest request = new CreateRequest()
+                .setEmail(registrationInfo.getEmail())
+                .setDisplayName(registrationInfo.getUsername())
+                .setPassword(registrationInfo.getPassword());
+
+        UserRecord userRecord;
+        try {
+            userRecord = FirebaseAuth.getInstance().createUser(request);
+        } catch (FirebaseAuthException e){
+            throw new RegistrationRuntimeException(e.getAuthErrorCode().name(), e);
```

Comment:
I think that we should differentiate errors that comes from firebase and throw suitable exception after that(Registartion could fail due to many different reasons) but maybe we should resolve this issue later, when we will create any global error handling.

Reply:
Okey, so I am leaving it unchanged for now.